sox.features.js

```diff
@@ -35,45 +35,57 @@
     },
 
     markEmployees: function () {
-      // Description: Adds an Stack Overflow logo next to users that *ARE* a Stack Overflow Employee
+      // Description: Adds the Stack Exchange logo next to users that *ARE* Stack Exchange employees
```

Comment:
Do you mind also updating this (the fact that it's the SE logo instead of SO) in the sox.features.json file?